test/unit/views/MucrOptionsViewSpec.scala

```diff
@@ -16,35 +16,33 @@
 
 package views
 
+import base.Injector
 import forms.MucrOptions
 import helpers.views.CommonMessages
 import org.jsoup.nodes.Document
 import play.api.data.Form
-import views.spec.UnitViewSpec
+import views.html.mucr_options
+import views.spec.{UnitViewSpec, ViewMatchers}
 
-class MucrOptionsViewSpec extends UnitViewSpec with CommonMessages {
+class MucrOptionsViewSpec extends UnitViewSpec with CommonMessages with ViewMatchers with Injector {
```

Comment:
This will need more discussion inside dev team, but there is also `ViewSpec`. It is better to use it as it inherits from `MessagesStub` mentioned earlier.

Reply:
I cannot findn any testst for errors here. Could you please add them?

Reply:
While we're having a discussion on message validation, @ed0906 created a "AllMessageKeysAreMandatoryMessages" in the internal service (which appears to not be "switched on" in movements) that will result in view specs failing if they render a page that tries to look up a message key that does not exist.

Should we try to use that in movements?   It saves having to write specific tests to assert that a message key is defined.

Reply:
Updated to use ViewSpec & added tests for errors